api/src/main/java/org/apache/iceberg/ContentFile.java

```diff
@@ -141,4 +141,13 @@
    *         null value counts, or nan value counts
    */
   F copyWithoutStats();
+
+  /**
+   * Returns the sort order id of this file, which describes how the file is ordered.
+   * This information will be useful for merging data and equality delete files more efficiently
+   * when they share the same sort order id.
+   */
+  default Integer sortOrderId() {
```

Comment:
Do we need a default?

Reply:
I was thinking to add it to ensure backward compatibility in case people implement their own `ContentFile`, do you think it's unlikely so it's safe for us to not have default?